README.md

```diff
@@ -3,19 +3,61 @@ Clapper
 
 Synchronize setup and deployment for the Director.
 
-Configuration Validation
-========================
+Ansible-based Validations
+-------------------------
+
+Recently, we focused on writing automated validations that require
+little human setup or interference. These are driven by
+[Ansible](http://ansible.com/) we plan to provide an API that will let
+us use these from the TripleO web UI and command line clients.
+
+### Running a validation
+
+For now, you need to run them manually. All the validations live in
+the [ansible-tests/validations](ansible-tests/validations) directory.
+
+To run a validation you need to:
+
+    $ git clone https://github.com/rthallisey/clapper.git
+    $ cd clapper/ansible-tests
+    $ ls validations  # pick a validation to run
+    $ ansible-playbook -v -i tripleo-ansible-inventory.py validations/some_validation.yaml
+
+NOTE: some validations will provide useful information that can cause
+trouble but isn't in an of itself enough to fail the check. Most
```

Comment:
It's the difference between:

```
$ ansible-playbook -i tripleo-ansible-inventory.py validations/rogue-dhcp.yaml
...
TASK: [Look for a rogue DHCP] ************************************************* 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=3    changed=1    unreachable=0    failed=0   
```

and:

```
$ ansible-playbook -v -i tripleo-ansible-inventory.py validations/rogue-dhcp.yaml
...
TASK: [Look for a rogue DHCP] ************************************************* 
changed: [localhost] => {"changed": true, "msg": "Found 1 DHCP servers.", "results": "DHCP servers: 192.168.122.1 (52:54:00:c2:dd:aa)"}

PLAY RECAP ******************************************************************** 
localhost                  : ok=3    changed=1    unreachable=0    failed=0
```

Currently, the rogue-dhcp validation can't tell whether a presence of a DHCP server is a problem or not so it just list's all and let the deployer decide. So, it'll never fail but it will return the dhcp servers in the task results. Problem is, unless you specify `-v`, you'll never see it.

Do you have an idea on how to write this more clearly?